meta-resin-common/recipes-containers/docker/docker/docker.conf.systemd

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon --log-driver=journald -s btrfs -H fd:// -H tcp://0.0.0.0:2375/
```

Comment:
Or if it is just for development, it would be okay I think. What is the use case for allowing docker daemon access on development images?